pom.xml

```diff
@@ -41,6 +41,21 @@
             <groupId>org.projectlombok</groupId>
             <artifactId>lombok</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.google.code.gson</groupId>
+            <artifactId>gson</artifactId>
+            <version>2.8.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>retrofit</artifactId>
+            <version>2.3.0</version>
+        </dependency>
+        <dependency>
+            <groupId>com.squareup.retrofit2</groupId>
+            <artifactId>converter-gson</artifactId>
+            <version>2.3.0</version>
+        </dependency>
```

Comment:
I went ahead and added dependencies for a retrofit client example, as I believe this will really help you when accessing external API's, and with your organization in the project.

Reply:
This is a good suggestion! However, we're not going to use Retrofit in this project.

Reply:
Avoid adding unimplemented features (even tests), as unused code is clutter, which is technical debt.